src/qml/main.qml

```diff
@@ -148,16 +148,10 @@ Application {
                     onClicked: layerStack.push(usbLayer)
                 }
                 ListItem {
-                    //% "Power Off"
-                    title: qsTrId("id-poweroff-page")
+                    //% "Power"*/
```

Comment:
Wupps, i had that commented out and forgot to remove the close comment *mental note*